simulator/tabletop.py

```diff
@@ -74,3 +89,26 @@ def start_app_on_all(self, app_type):
   def run(self):
     """ Runs the tabletop simulation indefinitely. """
     self.__canvas.wait_for_events()
+  
+  def draw_grid(self):
+    """ Creates Line objects for grid, returns the list of line objects in grid."""
```

Comment:
Formatting for docstring should be more like this:

Creates Line objects for grid
Returns:
  The list of line objects in the grid.